packages/hop-node/src/db/GasPricesDb.ts

```diff
@@ -51,7 +51,13 @@ class GasPricesDb extends BaseDb {
   }
 
   async getNearest (chain: string, targetTimestamp: number, staleCheck: boolean = true): Promise<GasPrice | null> {
-    const items : GasPrice[] = (await this.getItems()).filter((item: GasPrice) => item.chain === chain && item.timestamp)
+    const startTimestamp = targetTimestamp - OneHourSeconds
+    const endTimestamp = targetTimestamp + OneHourSeconds
+    const filter = {
+      gte: `${chain}:${startTimestamp}`,
+      lte: `${chain}:${endTimestamp}~`
+    }
+    const items : GasPrice[] = (await this.getItems(filter)).filter((item: GasPrice) => item.chain === chain && item.timestamp)
```

Comment:
Does it make sense to explicitly handle the case where `items` is `null` after this line? This might happen, for example, when, for some reason, there were no `gasPrice` db writes within +- 1 hour of this call. Another time may be when calling from the CLI.

Reply:
getItems will always return an array `[]` should it won't ever be null. `Promise.all` always returns an array